### DISCUSSION TITLE
Proper way to filter q-values in main report

### DISCUSSION THREAD

**Discussion by pisistrato** [vdemichev/DiaNN#710] [Q&A] [answered]
Hi Vadim,

After reading the documentation, issues and discussions, I still find a bit confusing what would be the proper way to filter the main report when doing library-free search (MBR on) for either a full proteome or a PTM dataset.
Are the following filtering criteria correct/enough?
1. Full proteome
- Lib.Q.Value < 0.01 & Lib.PG.Q.Value < 0.01
- optional but not advised, PG.Q.Value <= 0.01 to ensures run-specific protein FDR
- no need to do Q.Value < 0.01 ?

2. PTM
- PTM.Q.Value < 0.01
- no need to do Q.Value < 0.01, PTM.Q.Value already takes care of it?
- what about Lib.Q.Value < 0.01 ?

Thanks

**Comment by vdemichev** [comment, marked as answer]
1. Q.Value < 0.01 is indeed not necessary as it's automatically applied if the precursor FDR filter is set to 1% in the GUI, which is the default and recommended setting. Lib.Q.Value < 0.01 is likewise automatic with MBR in this case. So the only filter is Lib.PG.Q.Value.
2. PTM.Q.Value is optional, using MBR guarantees global PTM confidence filtering, while PTM.Q.Value is run-specific.

Best,
Vadim

**Reply by pisistrato** [reply]
Thanks for the fast reply.
So basically, in both cases Lib.PG.Q.Value would be enough.
Not sure it is possible, but it would be nice to have a sort of Lib.PTM.Q.Value in the table.

**Reply by vdemichev** [reply]
Lib.Q.Value filtering means automatic Lib.PTM.Q.Value filtering
